src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        Random randomColor = new Random();
```

Comment:
```suggestion
    private static final Random random = new Random();
    
    public Colors getRandomColor() {
        int index = random.nextInt(Colors.values().length);
```
lets make Random global